tests/integration_tests.py

```diff
@@ -0,0 +1,49 @@
+import requests
+
+
+class PetStore:
+    """Petstore class.
+
+    Attributes:
+        url: Petstore API base URL.
+    """
+    def __init__(self):
+        """Class constructor method."""
+        self.url = "http://localhost:80/pets"
+
+    def add_path_to_url(self, id):
+        """Return petstore base URL with appended path"""
+        return "/".join([self.url, str(id)])
+
+
+petstore = PetStore()
+
+
+def test_post_request():
+    """Test `POST /pets` for successfully adding a new pet."""
```

Comment:
Changed the docstring formats in line with the relevant PEP guidelines. Basically, start the comment right after the opening triple double quotes. If the entire comment, including opening and closing triple quotes, fits on one line (we use 79 chars for consistency even though docstrings should be even shorter, I think), then the closing triple quotes should go on that line, otherwise closing triple quotes should go on their own line.